using-mobile-extensions/offer-decisioning/api-reference.md

```diff
@@ -211,6 +211,160 @@ OfferDecisioning.retrievePrefetchedOffers(decisionScopes: [homeDecisionScope]) {
 ## Public classes
 
 {% tabs %}
+{% tab title="Android" %}
+### DecisionScope
+
+This class encapsulates the decision `activityId`, `placementId` and `itemCount`. The `DecisionScope` object can be used to fetch offers from the offer decisioning service if it is enabled in the Experience Edge network. The default value for `itemCount` is 1, if not provided when creating the `DecisionScope` object.
```

Comment:
from the offer decisioning service if it is enabled in the Experience Edge network -> `from the Offer Decisioning service when it is enabled in the Datastream configuration.`

Reply:
I meant to explain the significance of the given custom class in simple terms in the brief here. But if it's confusing, I'd like to update it as follows
...from the Offer Decisioning service when it is enabled in the Datastreams on the Data Collection UI.